dev-guide/host-port-registry.md

```diff
@@ -62,123 +62,122 @@ Ports are assumed to be used on all nodes in all clusters unless otherwise speci
-| 9258  | cluster-cloud-controller-manager-operator | cluster infra | 4.9 | metrics, control plane only |
-| 9300  | kube-proxy | sdn | 4.12 | metrics, ingress node firewall |
-| 9301  | kube-proxy | sdn | 4.12 | metrics, ingress node firewall |
-| 9444  | haproxy | sdn | 4.7 | on-prem internal loadbalancer, healthcheck port |
-| 9445  | haproxy | sdn | 4.7 | on-prem internal loadbalancer |
-| 9446  | baremetal-operator | metal | 4.9 | healthz; baremetal provisioning, control plane only |
-| 9447  | baremetal-operator | metal | 4.10 | webhook; baremetal provisioning, control plane only |
-| 9448  | run-once-duration-override-operator | workloads | 4.13 | webhook; run-once-duration-override |
-| 9537  | crio      | node || metrics |
-| 9641  | ovn-kubernetes northd | sdn | 4.3 | control plane only, ovn-kubernetes only |
-| 9642  | ovn-kubernetes southd | sdn | 4.3 | control plane only, ovn-kubernetes only |
-| 9643  | ovn-kubernetes northd | sdn | 4.3 | control plane only, ovn-kubernetes only |
-| 9644  | ovn-kubernetes southd | sdn | 4.3 | control plane only, ovn-kubernetes only |
-| 9978  | etcd      | etcd || metrics, control plane only |
-| 9979  | etcd      | etcd || ?, control plane only |
-| 9980  | etcd      | etcd || healthz, readyz |
-| 10010 | crio | node || stream port |
-| 10250 | kubelet | node || kubelet api |
-| 10251 | kube-scheduler | apiserver || healthz, control plane only |
-| 10255 | kube-proxy | sdn | 4.7 | healthz, third-party network plugins only |
-| 10256 | openshift-sdn | sdn || healthz, openshift-sdn only |
-| 10256 | kube-proxy | sdn || healthz, third-party network plugins only, deprecated |
-| 10257 | kube-controller-manager | apiserver || metrics, healthz, control plane only |
-| 10258 | cloud-controller-manager | cluster infra | 4.9 | metrics, healthz, control plane only |
-| 10259 | kube-scheduler | apiserver || metrics, control plane only |
-| 10263 | cloud-node-manager | cluster infra | 4.9 | metrics, healthz, some platforms only |
-| 10357 | cluster-policy-controller | apiserver || healthz, control plane only |
-| 10443 | haproxy   | net edge    | 3.0 | HAProxy internal `fe_no_sni` frontend; localhost only; baremetal only; only on nodes running router pod replicas |
-| 10444 | haproxy   | net edge    | 3.0 | HAProxy internal `fe_sni` frontend; localhost only; baremetal only; only on nodes running router pod replicas |
-| 17697 | kube-apiserver | apiserver || ?, control plane only |
-| 22623 | machine-config-server | node || control plane only |
-| 22624 | machine-config-server | node || control plane only |
-| 60000 | baremetal-operator | metal || metrics, 4.6+, control plane only |
+| Port  | Process   | Protocol | Control-plane only | Owning Team | Since | Notes |
+|-------|-----------|----------|--------------------|-------------|-------|-------|
+| 80    | haproxy   ||| net edge    | 3.0 | HTTP routes; baremetal only; only on nodes running router pod replicas |
```

Comment:
Possibly worth using a column like `Nodes` with values like `control-plane`, `nodes running router pod replicas`, `all`, etc., instead of using the `Control-plane only` (`yes`/`no`) I'm currently floating here.

Reply:
I don't think there are many situations where we run hostNetwork pods that listen on ports (on non-control-plane nodes).